nikola/data/themes/base/templates/base.tmpl

```diff
@@ -20,7 +20,13 @@ ${template_hooks['extra_head']()}
          ${footer.html_footer()}
     </div>
     ${base.late_load_js()}
+    % if use_cdn:
+        <script src="//cdnjs.cloudflare.com/ajax/libs/baguettebox.js/1.8.2/baguetteBox.min.js"></script>
```

Comment:
protocol-relative stopped being cool [a while back](https://www.paulirish.com/2010/the-protocol-relative-url/). Using httpS never gave mixed-content warnings, only http did.